modules/ROOT/pages/logs-using.adoc

```diff
@@ -16,6 +16,9 @@ Logs produced by apps with customized logging (by changing the pattern layout in
 * `<PatternLayout pattern="%-5p %d [%t] [event: %X{correlationId}] %c: %m%n" />`, or
 * `<PatternLayout pattern="%d [%t] %-5p %c - %m%n" />`
 
+==== Note For Applications Deployed to Cloudhub and Runtime Fabric With Customized Time Zone
+Do not override the default JVM timezone(it should always be UTC). Changing the timezone to a local timezone will break Anypoint Monitoring.
```

Comment:
`Changing the timezone will cause logs indexed into Anypoint Monitoring with incorrect timestamp.`